src/components/Text.js

```diff
@@ -4,14 +4,13 @@ import classNames from 'classnames'
 
 import { fontSizes } from '../theme'
 
-// TODO: Should we have a block level version which uses a 0 margin p?
```

Comment:
Interestingly I would keep this TODO in as I think it was left by a previous dev I think. I tend to have a rule where I don't delete other devs TODOs unless I've completed them or they are happy for me to.